frontend/src/queries/useBakerQuery.ts

```diff
@@ -80,6 +80,11 @@ const BakerQuery = gql<BakerResponse>`
 							rank
 							total
 						}
+						paydayCommissionRates {
+							transactionCommission
+							finalizationCommission
+							bakingCommission
```

Comment:
This will be called Block reward commission for the users.

Reply:
Rendering updated but existing code fields and classes not updated (as like comment above).